test/validation/api/classification/odp_classification_basic.c

```diff
@@ -147,6 +147,10 @@ static void cls_create_cos_multi(void)
 
 	num = capa.max_cos < MAX_HANDLES ? capa.max_cos : MAX_HANDLES;
 
+	param = calloc(MAX_HANDLES, sizeof(odp_cls_cos_param_t));
+	if (param == NULL)
+		return;
```

Comment:
Only `num` params is required and `calloc` shouldn't normally fail, so instead of silently skipping the test I'd fail here. E.g. `CU_ASSERT_FATAL(param != NULL);`

Otherwise the PR is looking OK.